LICENSE.txt

```diff
@@ -0,0 +1 @@
+Copyright (c) 2024 DEP-12. All Rights Reserved.
```

Comment:
This is not enough.

Reply:
[6ac95a7](https://github.com/Yasith-s-play-ground/pull-request-demo/pull/1/commits/6ac95a7d4a9f4cc22ee78ac20a80ce94d6c00acd)